python/triton/testing.py

```diff
@@ -108,6 +126,7 @@ def do_bench_cudagraph(fn, rep=20, grad_to_none=None, quantiles=None, return_mod
                 if grad_to_none is not None:
                     for x in grad_to_none:
                         x.grad = None
+                maybe_clear_cache()
                 fn()
```

Comment:
It's a bit weird to have the cache flushing time included in the benchmark measurement. I suppose for autotuning purposes it should be fine as all calls are effected the same way, but there should at least be a warning in the doc string.